renderdoc/driver/gl/wrappers/gl_buffer_funcs.cpp

```diff
@@ -2960,6 +2960,25 @@ bool WrappedOpenGL::Serialise_glVertexArrayVertexAttribOffsetEXT(GLuint vaobj, G
   return true;
 }
 
+std::function<void()> WrappedOpenGL::ConvertClientMemoryPointer(GLuint index, const void *&pointer)
+{
+  GLuint buffer;
+  m_Real.glGetIntegerv(eGL_VERTEX_ARRAY_BINDING, (GLint *)&buffer);
```

Comment:
This is a bit confusing - I'd prefer `GLuint vao` since it's not a buffer we're querying but a buffer we're binding.